code/modules/mob/living/carbon/carbon.dm

```diff
@@ -666,6 +666,8 @@
 	//Fire and Brute damage overlay (BSSR)
 	var/hurtdamage = getBruteLoss() + getFireLoss() + damageoverlaytemp
 	if(hurtdamage)
+		if(has_trait(TRAIT_NUMB))
+			hurtdamage = max((hurtdamage - 20) / 2, 0) //Make the damage appear smaller than it really is

```

Comment:
people threw a fit over miner's salve hiding the damage amount

Reply:
what, when? i remember complaints about the instastun, but i don't think hiding damage was ever an issue